packages/server/utils/AtlassianServerManager.ts

```diff
@@ -1,8 +1,8 @@
 import fetch from 'node-fetch'
 import AtlassianManager from 'parabol-client/utils/AtlassianManager'
 import makeAppURL from 'parabol-client/utils/makeAppURL'
+import {authorizeOAuth2} from '../integrations/helpers/authorizeOAuth2'
 import appOrigin from '../appOrigin'
-import {OAuth2Error, OAuth2Success} from '../types/custom'
 
 interface AuthQueryParams {
   grant_type: 'authorization_code'
```

Comment:
this is defined elsewhere, feels bad having the same defs floating around under different names